automation/docker/application.conf.ctmpl

```diff
@@ -3,6 +3,7 @@
 {{with $environment := env "ENVIRONMENT"}}
 {{with $dir := env "ROOT_DIR"}}
 {{with $local_ui := env "LOCAL_UI"}}
+{{with $trialBillingPemSecrets := vault (printf "secret/dsde/firecloud/%s/common/trial-billing-account.pem" $environment)}}
```

Comment:
Most of these additions are not used by Leonardo automation, but they are required by the current iteration of workbench-libs automation config.  Removing these is a TODO but let's not block on it.

Reply:
should vault-y confs be living in a public repo?

Reply:
every repo will be getting this